app/forms/submissions/base_validator.rb

```diff
@@ -22,6 +22,7 @@ def validator(name, options)
     validator :schools, form: "Schools::FormValidator", if: :requires_schools?
     validator :funding, form: "Funding::FormValidator", if: :requires_funding?
     validator :iqts_country, form: "IqtsCountryForm", if: :requires_iqts_country?
+    validator :placements, form: "PlacementDetailForm", if: :requires_placements?
```

Comment:
We'll need to use a composite validator here to check `PlacementDetailForm` and the one Steve is building. Just making a note here so we don't forget!

Reply:
Added to snag list